### ISSUE TITLE
Resource Report - 2.1.1 Keyboard - None of the subsection title chevrons are accessible using the keyboard.

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9765]
As a user reliant on the keyboard to navigate and select things on the page, I should be able to access all components in the page which are accessible/selectable using a mouse pointer, using a keyboard.

**Linked Work Item:**
#9389 

**Expected Result:**
All components in the page which are accessible using the mouse, including the subtitle chevrons, should be accessible using the keyboard.

**Actual Result:**
None of the subtitle chevrons are keyboard accessible - the keyboard only focuses on the links underneath each chevron. 

**Evidence:**

It isn't quite clear from the video because there are no focus indicators but selecting something within the page actually only takes you to another linked report - the chevrons aren't available at all.

![Image](https://github.com/archesproject/arches/assets/77161085/d07fd35f-a83a-48b0-bc18-d10c56230210)

**Comment by johnatawnclementawn** [comment]
This will be addressed by #9544

**Comment by johnatawnclementawn** [comment]
Resolved by #9544

**Comment by imogenroberts** [comment]
RETEST: Resource report subsection chevrons are now accessible using the keyboard, and can be expanded or collapsed by pressing Enter.

**Evidence:**


![Image](https://github.com/archesproject/arches/assets/77161085/b7ffdc30-c33e-4088-985c-39e9cd849431)
